backend/pong/consumers.py

```diff
@@ -40,15 +40,15 @@ async def connect(self):
                 self.game.use_ai_opponent()
                 asyncio.create_task(self.ai_opponent_loop())
                 asyncio.create_task(self.ai_move_loop())
-            self.start_match(self.match)
+            await self.start_match(self.match)
             await self.accept()
             asyncio.create_task(self.send_positions_loop())
         else:
             await self.close()
             return
 
     async def disconnect(self, close_code):
-        await sync_to_async(self.match.abort_match)()
```

Comment:
Was this intentional? We should wrap this non/asyncronous function with the sync_to_async.